src/api/compiler.ts

```diff
@@ -0,0 +1,24 @@
+import { BaseApi, extendsProfile } from "./base"
+import { ModuleProfile, Api, API } from "src/types"
+
+export interface ICompilerApi extends Api {
+  events: {
+    compilationFinished: []
```

Comment:
is it possible to define the parameters here: 
`file: string, source: {sources:<map of files>, target: string}} , languageVersion: string, data: compilationResult ` ?
target is the same as file and compilationResult should be https://solidity.readthedocs.io/en/v0.5.7/using-the-compiler.html#output-description
(but perhaps a bit too much to type all that)

Reply:
Sure ! This is the goal of this interface. I just didn't know where to find the types. Thanks for the documentation.